File: isa/csharp/ChimeraISA.cs
namespace Chimera.ISA;
public static class ChimeraISA {
    public const string CatalogVersion = "CHM-ISA-CATALOG-1";
    public const string CatalogPath = "isa/catalog.json";
    public readonly record struct InstructionRef(string Id, string Family, string Mnemonic, string Syntax, string Hex);
}

File: isa/kotlin/ChimeraISA.kt
package chimera.isa
object ChimeraISA {
    const val CATALOG_VERSION = "CHM-ISA-CATALOG-1"
    const val CATALOG_PATH = "isa/catalog.json"
    data class InstructionRef(val id: String, val family: String, val mnemonic: String, val syntax: String, val hex: String)
}
